get_all_manifests.sh

```diff
@@ -49,21 +49,40 @@ fi
 TMP_DIR=$(mktemp -d -t "odh-manifests.XXXXXXXXXX")
 trap '{ rm -rf -- "$TMP_DIR"; }' EXIT
 
+function git_fetch_ref()
+{
+
+    local repo=$1
+    local ref=$2
+    local dir=$3
+    local git_fetch="git fetch -q --depth 1 $repo"
+
+    mkdir -p $dir
+    pushd $dir &>/dev/null
+    git init -q
+    # try tag first, avoid printing fatal: couldn't find remote ref
+    if ! $git_fetch refs/tags/$ref 2>/dev/null ; then
+        $git_fetch refs/heads/$ref
+    fi
+    git reset --hard FETCH_HEAD
```

Comment:
```suggestion
    git reset -q --hard FETCH_HEAD 
```
to suspend checkout commit

Reply:
> to suspend checkout commit

As you wish :)